### ISSUE TITLE
(Suggestion) About:Blank Cloaking

### ISSUE THREAD

**Issue by Thisisanalthowcool** [Blobby-Boi/blobrowser#1]
It would be nice to have the browser open in about:blank, so combined with the extension hanger everything would be unblocked and nothing would show in history with the blob browser. A little refining would be very nice too!

**Comment by Blobby-Boi** [comment]
ok thx for the feedback!
